### PR TITLE
Add ability to add Keycloak specific ingress annotations

### PR DESCRIPTION
**Pull request by Kidswiss**
With this it's possible to set arbitrary annotations on the ingress objects for Keycloak.

## Checklist

- [x] The PR has a meaningful title. It will be used to auto generate the
      changelog.
      The PR has a meaningful description that sums up the change. It will be
      linked in the changelog.
- [x] PR contains a single logical change (to build a better changelog).
- [x] Categorize the PR by adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.
- [x] Link this PR to related issues or PRs.

### CODE REVIEW

File: component/class/defaults.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ parameters:
       appcat:
         registry: ghcr.io
         repository: vshn/appcat
-        tag: v4.61.0
+        tag: keycloak/staging_le
```
Comment (Kidswiss) [inline component/class/defaults.yml:48]: Will be adjusted

File: package/main.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@ parameters:
     image:
       registry: ghcr.io
       repository: vshn/appcat
-      tag: v4.61.0
+      tag: keycloak/staging_le
```
Comment (Kidswiss) [inline package/main.yaml:10]: Here as well

**Review by wejdross — APPROVED** [review]
this one is just a formality - LGTMT

**Comment by Kidswiss** [comment]
> this one is just a formality - LGTMT

You did notice the changes in the e2e tests... right?

**Pull request merged**